mne/channels/channels.py

```diff
@@ -257,7 +257,7 @@ def get_montage(self):
         # get the channel names and chs data structure
         ch_names, chs = self.info['ch_names'], self.info['chs']
         picks = pick_types(self.info, meg=False, eeg=True,
-                           seeg=True, ecog=True)
+                           seeg=True, dbs=True, ecog=True)
```

Comment:
put the dbs at the end not just after seeg everywhere.

Reply:
Thanks, will do!